js/models/conversations.js

```diff
@@ -1848,6 +1852,7 @@
 
       const updateParams = {
         timestamp: Date.now(),
+        identifier: messageId,
```

Comment:
Same as the comment below, we also sync this manually so we have to be careful that we don't sync this twice

Reply:
fixed